tests/benchmarks/test_cvrp_augerat.py

```diff
@@ -36,7 +35,18 @@ def test_subproblem_cspy(self):
         self.data.solve(cspy=True)
         assert round(self.data.best_value, -1) in [450, 460]
 
-    def test_subproblem_lp_with_initial_routes(self):
+    def test_subproblem_lp_dive(self):
+        self.data.solve(dive=True)
+        assert round(self.data.best_value, -1) in [450, 460]
+
+    def test_subproblem_cspy_dive(self):
+        self.data.solve(cspy=True, dive=True)
+        assert round(self.data.best_value, -1) in [450, 460]
+
+    def test_master_dive(self):
+        self.data.solve(dive=True)  #?
+
+    def test_subproblem_lp_with_initial_routes_dive(self):
```

Comment:
Add both the old test (with no diving) as well as this new one with `dive=True`.
To make sure they are both working